redbot/core/utils/_internal_utils.py

```diff
@@ -24,6 +26,8 @@
 main_log = logging.getLogger("red")
 
 __all__ = ("safe_delete", "fuzzy_command_search", "format_fuzzy_results", "create_backup")
```

Comment:
Could you add the missing `send_to_owners_with_preprocessor`, `send_to_owners_with_prefix_replaced` and newly added `expected_version` to `__all__`?